### PR TITLE
Fix lstm and add rnn, gru

### PR DESCRIPTION
**Pull request by fumihwh**
onnx test cases have some problem. I will fix that also.

### CODE REVIEW

**Comment by tjingrant** [comment]
OK thanks! Looks like enormous amount of work; I will review promptly once you request me to!

**Comment by tjingrant** [comment]
(BTW, I assume you want to merge after fixing the onnx tests? or do you want to merge before that, in which case I should start reviewing ASAP?)

**Comment by fumihwh** [comment]
@tjingrant 
Yes, a PR was created to fix the lstm test cases in ONNX.
You can pull that if you want to check this PR works or not.

File: onnx_tf/backends/backend_v1.py [hunk 1]
```diff
@@ -638,67 +639,158 @@ def handle_l_r_n(cls, node, input_dict):
...
+      cell_kwargs["forget_bias"] = 0.
+      cell_kwargs["num_units"] = hidden_size
+      initial_state = None
+      initial_state_bw = None
+      if input_size >= 7 and node.inputs[5] in input_dict and node.inputs[6] in input_dict:
+        initial_state = (tf.nn.rnn_cell.LSTMStateTuple(
+            input_dict[node.inputs[6]][0], input_dict[node.inputs[5]][0]),)
+        if num_directions == 2:
+          initial_state_bw = initial_state = (tf.nn.rnn_cell.LSTMStateTuple(
+              input_dict[node.inputs[6]][1], input_dict[node.inputs[5]][1]),)
```
Comment (tjingrant) [inline onnx_tf/backends/backend_v1.py]: Can we have more meaningful variable names than inputs[6]? 
Maybe we can extract these tensors out to variables like input_forget.

**Comment by fumihwh** [comment]
Refactored and add RNN also.

**Comment by fumihwh** [comment]
GRU added.

**Pull request merged**